use-case-guides/concur-expense-report-lookup/README.md

```diff
@@ -0,0 +1,143 @@
+
+## Creator Studio Components
+
+- **Triggers:**
+    1. Natural Language
+- **Slots**:
+    1. None
+- **Actions:**
+    1. Query Concur Expense Reports for the User
+- **Guidelines:**
+    1. None
+
+Based on the needs of this use case, we should build a [Lookup Single Record by ID / Keyword](https://developer.moveworks.com/creator-studio/design-patterns/dp-6/)**.**
+
+## API Research
+
+There’s only 1 API needed to build this use case
+
+### API #1: Expense Report Lookups
+
+Since we want to integrate with Concur Expense, we should look into their [Expense v3 APIs](https://developer.concur.com/api-reference/expense/expense-report/v3.reports.html). For this use case, we will need only 1 API call.
+
+![Untitled](Use%20Case%20Tutorial%20Lookup%20Expense%20Reports%20in%20Concur%200ad48c0ab26047b1bab45a82557a0bda/Untitled.png)
+
+1. [Expense v3 APIs](https://developer.concur.com/api-reference/expense/expense-report/v3.reports.html): An API for Concur to retrieve expense reports owned by the user.
+    
+    Here is the API we used to fetch 10 expense reports for the given user’s email address
+    
+    ```bash
+    curl --location 'https://us2.api.concursolutions.com/api/v3.0/expense/reports?user={{user_email_address}}&limit=10' \
+    --header 'Accept: application/json' \
+    --header 'Authorization: Bearer {{generated_bearer_token}}'
+    ```
+    
+
+> In the above API, we have used email address as the user identifier because during User setup in Concur the `Use Email as Username` checkbox was checked out.
+
+This is required for building this use case since we will be leveraging the User’s email address from the [User Attributes](https://developer.moveworks.com/creator-studio/reference/user-attributes/#user-attributes-reference) mapping in Creator Studio. This email address is picked up from the address that is mapped for the Chat System that the User is using it on.
+For example, if `user@example.com` is mapped in the Chat System, then the use case will only work if the same `user@example.com` was used in your Concur instance.
```

Comment:
Some callout like: "if you don't do this, you'll need to figure out where you store email address -> Coupa User ID mappings"